package.json

```diff
@@ -22,9 +22,10 @@
   },
   "homepage": "https://github.com/massiveart/eslint-config-ma#readme",
   "devDependencies": {
-    "eslint": "^4.19.1"
+    "eslint": "^8.37.0",
+    "eslint-plugin-import": "^2.27.5"
```

Comment:
this should be a normal dependency like stylelint prepend: https://github.com/massiveart/stylelint-config-ma/blob/c696964e740ce65776a6629708a84bc0fad0d508/package.json#L25

Reply:
@alexander-schranz Thanks, done!